contracts/src/v0.8/dev/BatchBHS.sol

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.6;
+
+/**
+ * @title BatchBHS
+ * @notice The BatchBHS contract acts as a proxy to write many blockhashes to the
+ *   provided BlockhashStore contract efficiently in a single transaction. This results
+ *   in plenty of gas savings and higher throughput of blockhash storage, which is desirable
+ *   in times of high network congestion.
+ */
+contract BatchBHS {
```

Comment:
Should this be called BatchBlockhashStore just to match the original BlockhashStore? The name doesn't seem to long to me

Reply:
Done.